src/main/java/quest/com/controller/AuthFilter.java

```diff
@@ -0,0 +1,49 @@
+package quest.com.controller;
+
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.annotation.WebFilter;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import java.io.IOException;
+
+import static java.util.Objects.isNull;
+
+@WebFilter(filterName = "AuthFilter")
+public class AuthFilter implements Filter {
+    public void init(FilterConfig config) throws ServletException {
+    }
+
+    public void destroy() {
```

Comment:
do we need this method here? it seems that in Filter we have default implementation